packages/cli/test/fixtures/unit/edge-middleware-strict/_middleware.ts

```diff
@@ -0,0 +1,6 @@
+export default function (req) {
+  const isStrict = (function () {
+    return !this;
+  })();
+  return new Response('is strict mode? (next.js) ' + (isStrict ? 'yes' : 'no'));
```

Comment:
Why `(next.js)`?

Reply:
No reason, copy/pasta'd from the original report that strict mode wasn't in place.